Mage.Sets/src/mage/cards/a/AllWillBeOne.java

```diff
@@ -0,0 +1,94 @@
+package mage.cards.a;
+
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.dynamicvalue.common.SavedDamageValue;
+import mage.abilities.effects.common.DamageTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.TargetController;
+import mage.constants.Zone;
+import mage.filter.common.FilterCreaturePlayerOrPlaneswalker;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.target.common.TargetAnyTarget;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class AllWillBeOne extends CardImpl {
+
+    public AllWillBeOne(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{3}{R}{R}");
+
+        // Whenever you put one or more counters on a permanent or player, All Will Be One deals that much
+        // damage to target opponent, creature an opponent controls, or planeswalker an opponent controls.
+        this.addAbility(new AllWillBeOneTriggeredAbility());
+    }
+
+    private AllWillBeOne(final AllWillBeOne card) {
+        super(card);
+    }
+
+    @Override
+    public AllWillBeOne copy() {
+        return new AllWillBeOne(this);
+    }
+}
+
+class AllWillBeOneFilter extends FilterCreaturePlayerOrPlaneswalker {
```

Comment:
this doesn't need to be a separate class, just make a static `FilterCreaturePlayerOrPlaneswalker` in the triggered ability class and modify it

Reply:
Done